code/datums/components/crafting/crafting.dm

```diff
@@ -214,10 +214,10 @@
 				if(result.atom_storage && recipe.delete_contents)
 					for(var/obj/item/thing in result)
 						qdel(thing)
-			result.reagents?.clear_reagents()
 			var/datum/reagents/holder = locate() in parts
 			if(holder) //transfer reagents from ingredients to result
-				if(result.reagents)
+				if(!ispath(recipe.result,  /obj/item/reagent_containers) && result.reagents)
```

Comment:
No. This will undeniably cause issues for certain recipes that should inherit the components' reagents despite being reagent containers, like molotovs.
A better solution should be found. Perhaps remove the ethanol and readd the lizard wine reagent on the `CheckParts` for that object.

Reply:
It won't cause issues because molotovs override check parts proc and transfer the reagents themselves from the components.

https://github.com/tgstation/tgstation/blob/f40d471712e739065278c5aa088f74d6ce8af1a1/code/modules/reagents/reagent_containers/cups/glassbottle.dm#L787

We could write an extra check to still transfer reagents if the container is empty.

Reply:
We could write an extra check to still transfer reagents if the container is empty. Not sure when that would be required though

Reply:
Mhm, good to know. Case dismissed.